libkbfs/block_retrieval_queue.go

```diff
@@ -163,8 +169,84 @@ func (brq *blockRetrievalQueue) notifyWorker() {
 	}
 }
 
+func (brq *blockRetrievalQueue) CacheAndPrefetch(ptr BlockPointer, block Block,
```

Comment:
I just realized this is already commented in `interfaces.go`.  Maybe you could add a note about the LRU time there?  It's just not obvious from the name that that happens.